install/kubernetes/helm/istio/charts/kiali/templates/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: kiali
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: kiali
+spec:
+  type: NodePort
+  ports:
+  - name: tcp
+    protocol: TCP
+    port: 20001
```

Comment:
Same thing here, it should be configurable.

Reply:
The container port is fine hardcoded - but in particular for node port it must be customized, others may use that port. But see above, better to remove the node port instead.

Reply:
Removed NodePort